### ISSUE TITLE
[desktop] None of the example notebooks can be executed

### ISSUE THREAD

**Issue by rgbkrk** [nteract/archived-desktop-app#3335]
The example notebooks are awesome. They're packed in the [asar archive](https://electronjs.org/docs/tutorial/application-packaging) since they're a dependency desktop relies on. Real easy to open. We've got a tricky little problem now though. Notebooks open with a current working directory where the notebook is located.

![screen shot 2018-09-17 at 9 06 45 pm](https://user-images.githubusercontent.com/836375/45666335-f5cf5a80-bac9-11e8-81b9-14840ee04bef.png)

Since it's not a real path, the javascript kernel errors for some of the `fs` operations that aren't supported.

Somehow I think we have to force `@nteract/examples` to not end up in the asar. It's tempting to just disable asar, though it's super helpful for windows (due to nested paths).

**Comment by lgeiger** [comment]
You can use [`asarUnpack`](https://www.electron.build/configuration/configuration#overridable-per-platform-options) in the builder config to ignore specific dependencies from being packed.

**Comment by rgbkrk** [comment]
Hmmm... I tried that one before yet it seems maybe I didn't apply it right. It looks like I have to put it in each of `windows`, `linux`, and `mac`.

**Comment by rgbkrk** [comment]
Ah no, just like last time I tried this, it didn't even accept the option:

```
Error: Configuration is invalid.
 - configuration.asar should be one of these:
   object { externalAllowed?, ordering?, smartUnpack? } | {
     "type": [
       "null",
       "boolean"
     ]
   }
```

**Comment by rgbkrk** [comment]
Fixed on the last release via #3477
